### ISSUE TITLE
Error deploying docker on Ubuntu 18.04

### ISSUE THREAD

**Issue by jamesgibson14** [zodern/meteor-up#944]
Probably related to https://unix.stackexchange.com/questions/363048/unable-to-locate-package-docker-ce-on-a-64bit-ubuntu

**Comment by systemlevel** [comment]
Mup deploy does not work on 18.04 for me either.  

When it deploys I get the following errors during verification:

 0curl: (7) Failed to connect to 172.17.0.2 port 80: Connection refused

	Container has no IP Address, likely from it restarting.
	Container has no IP Address, likely from it restarting.
	Container has no IP Address, likely from it restarting.

When I deploy on Ubuntu 16.04 everything works.  

Anyone else experiencing these issues on ubuntu 18.04?  After searching forums and google for hours I still have not found a solution to get it to work on 18.04 bionic.

**Comment by marcoschwartz** [comment]
Same here, however I am having the same error but with Ubuntu 16.04

**Comment by zodern** [comment]
@marcoschwartz could you please show the output from the command?

**Comment by marcoschwartz** [comment]
Exact same as @systemlevel: 0curl: (7) Failed to connect to 172.17.0.2 port 80: Connection refused

Just as a note, after a clean install of Ubuntu 16.04 (on digital ocean, the DB is on MongoLab), the error is gone. So probably something coming from some more recents packages installed when doing apt-get update & upgrade.

**Comment by rashmimhatre100** [comment]
In our case deployment is not happening it is stuck at extracting tools or sometimes at Building for web.browser.legacy. 

But for one of our project we are using AWS ubuntu 18.04 only and were facing issue,
**Container has no IP Address, likely from it restarting.**

Tried stopping/removing mongodb conatiners, re-running it using docker.
Then we checked the mongodb shell version it was different than configured in mup.js and observed version applied in mup.js and somehow it worked.

Tried to implement above in different project but didn't worked, mup deploy is stuck.
